support-metrics-common/src/main/java/io/confluent/support/metrics/BaseMetricsReporter.java

```diff
@@ -160,79 +163,57 @@ protected boolean sendToConfluentEnabled() {
   public void run() {
     try {
       if (reportingEnabled()) {
-        boolean terminateEarly = waitForServer();
-        if (terminateEarly) {
-          log.info("Metrics collection stopped before it even started");
-        } else {
-          log.info("Starting metrics collection from monitored component...");
-          while (!Thread.currentThread().isInterrupted()) {
-            submitMetrics();
-            Thread.sleep(Jitter.addOnePercentJitter(reportIntervalMs));
-
-          }
+        waitForServer();
+        while (!isClosing) {
+          submitMetrics();
+          Thread.sleep(Jitter.addOnePercentJitter(reportIntervalMs));
```

Comment:
Can we add a log.info statement here, similar to: https://github.com/confluentinc/support-metrics-common/pull/45/files#diff-f0c3c736323a4e8e6e08f2509444641aR200. Thanks!